doc/widgets/scraper.rst

```diff
@@ -0,0 +1,42 @@
+Preprocess
+==========
+
+.. figure:: icons/Scraper.svg
+
```

Comment:
I would suggest you to include a simple .png icon file as elsewhere in the documentation. Svg doesn't get rendered well in the documentation.